ci-operator/step-registry/ovn/conf/custom-cidr/ovn-conf-custom-cidr-ref.yaml

```diff
@@ -4,12 +4,12 @@ ref:
   commands: ovn-conf-custom-cidr-commands.sh
   resources:
     requests:
-      cpu: 10m
-      memory: 100Mi
+      cpu: 1000m
+      memory: 1Gi
   env:
   - name: CLUSTER_NETWORK_CIDR
-    default: "10.128.0.0/20"
+    default: "10.128.0.0/14"
```

Comment:
Please do not change the default here because that will impact other jobs using this step. Instead, please customize the CLUSTER_NETWORK_CIDR and CLUSTER_NETWORK_HOST_PREFIX as env in your own job.